openfl-tutorials/experimental/Keras_MNIST_with_CPU.ipynb

```diff
@@ -0,0 +1,320 @@
+{
+ "cells": [
```

Comment:
I suggest adding an overview cell, which summarizes the goal and the main steps of the tutorial.

Reply:
Added steps and descriptions to cells where it applies.